test/template.yaml

```diff
@@ -6,6 +6,8 @@ objects:
   apiVersion: v1
   metadata:
     name: test
+    annotations:
+      ci-operator.openshift.io/container-sub-tests: test
```

Comment:
Is it possible for us to write a test that would show us this behavior actually happening?

Reply:
Probably.